### PR TITLE
Add caddyfile path for default config

### LINKED ISSUE #78: Custom config without docker swarm
What do you thing about adding a way to merge a default Caddyfile that we can mount in container so we can use this without swarm ?

This will allow to add default config like redirect http to https.

### PR DESCRIPTION
**Pull request by Its-Alex**
Add cli command and env variable to use a default CaddyFile

Fix #78

### CODE REVIEW

**Comment by Its-Alex** [comment]
@lucaslorentz I can't make it works with go 1.11.1, can you provide some advices to test it ? Thanks

**Comment by Its-Alex** [comment]
Up ?

**Comment by lucaslorentz** [comment]
LGTM

File: plugin/generator.go [hunk 1]
```diff
@@ -105,6 +117,19 @@ func (g *CaddyfileGenerator) GenerateCaddyFile() []byte {
 
 	directives := map[string]*directiveData{}
 
+	if g.caddyFilePath != "" {
+		dat, err := ioutil.ReadFile(g.caddyFilePath)
+		if err != nil {
+			if _, err := buffer.Write(dat) {
```
Comment (crvv) [inline plugin/generator.go]: This line can't compile.
Do you mean `if _, err := buffer.Write(dat); err {}` ?

Comment (Its-Alex) [reply plugin/generator.go]: Yes, thanks! I made a PR to solve it #82

**Pull request merged**